src/shared/components/comment/comment-form.tsx

```diff
@@ -84,15 +84,14 @@ export class CommentForm extends Component<CommentFormProps, any> {
       : capitalizeFirstLetter(I18NextService.i18n.t("reply"));
   }
 
-  handleCommentSubmit(content: string, form_id: string, language_id?: number) {
+  handleCommentSubmit(content: string, _form_id: string, language_id?: number) {
```

Comment:
Is there any reason this form ID argument wasn't removed entirely?

Reply:
MarkdownTextArea wants the function signature to contain 3 parameters in that order. language_id is needed to pass to the server.

Reply:
You can probably just remove that from markdowntextarea too, since I don't think its necessary anymore.